### PR TITLE
[AAE-5843] - Add e2e method to check the visibility of the upload fro…

### PR DESCRIPTION
**Pull request by arditdomi**
…m local tab

**Please check if the PR fulfills these requirements**

> - [X] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [X] Other... Please describe: e2e 

**What is the current behaviour?** (You can also link to an open issue here)
https://alfresco.atlassian.net/browse/AAE-5843

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [X] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/testing/src/lib/protractor/content-services/dialog/content-node-selector-dialog.page.ts [hunk 1]
```diff
@@ -189,4 +190,10 @@ export class ContentNodeSelectorDialogPage {
     contentListPage(): DocumentListPage {
         return this.contentList;
     }
+
+    async checkUploadFromLocalTabIsDisabled() {
+        await BrowserVisibility.waitUntilElementIsPresent(this.uploadFromLocalTab);
+        const disabled = await BrowserActions.getAttribute(this.uploadFromLocalTab, 'aria-disabled');
+        expect(disabled).toBe('true');
```
Comment (sivakumar414ram) [inline lib/testing/src/lib/protractor/content-services/dialog/content-node-selector-dialog.page.ts:197]: I am not sure if the page object is allowed to have expect function. Generally the real test should test the expected result.  Can we double check please?

**Review by sivakumar414ram — COMMENTED** [review]
Please check the comment

File: lib/testing/src/lib/protractor/content-services/dialog/content-node-selector-dialog.page.ts (hunk 1 above)
Comment (arditdomi) [reply lib/testing/src/lib/protractor/content-services/dialog/content-node-selector-dialog.page.ts:197]: There are places that we have expectations inside the page like here https://github.com/Alfresco/alfresco-ng2-components/blob/develop/lib/testing/src/lib/protractor/content-services/pages/search/date-range-filter.page.ts But I don't know to be honest, if it's a good practice or not

Comment (adinapitul) [reply lib/testing/src/lib/protractor/content-services/dialog/content-node-selector-dialog.page.ts:197]: @sivakumar414ram  is right, it is not a good practice to have expects in the page objects.

Comment (arditdomi) [reply lib/testing/src/lib/protractor/content-services/dialog/content-node-selector-dialog.page.ts:197]: Thank you both, I will update the expectation to be in the test 🙂

**Review by sivakumar414ram — APPROVED** [review]

**Pull request merged**